modular_skyrat/modules/goofsec/code/clothing/security_head.dm

```diff
@@ -0,0 +1,67 @@
+/**
+ * Standard security helmet
+*/
+/obj/item/clothing/head/helmet/sec
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/hats.dmi'
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/head.dmi'
+	icon_state = "security_helmet"
+	toggle_message = "You pull the visor down on"
+	alt_toggle_message = "You push the visor up on"
+	actions_types = list(/datum/action/item_action/toggle)
+	can_toggle = TRUE
+	toggle_cooldown = 0
+	flags_cover = HEADCOVERSEYES | HEADCOVERSMOUTH | PEPPERPROOF
```

Comment:
with this as it is currently, the helmet will provide eye/mouth/pepper protection at all times, rather than just when the visor is down. is that intentional?